src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -167,25 +167,20 @@ def _validate_secrets(secrets, os_type):
 
 # region VM Create Validators
 
+
 def _parse_image_argument(cmd, namespace):
     """ Systematically determines what type is supplied for the --image parameter. Updates the
         namespace and returns the type for subsequent processing. """
     from msrestazure.tools import is_valid_resource_id
     from msrestazure.azure_exceptions import CloudError
+
     # 1 - easy check for URI
     if namespace.image.lower().endswith('.vhd'):
         return 'uri'
 
-    # 2 - attempt to match an URN alias (most likely)
```

Comment:
I re-ordered the poking logics so to avoid pulling alias doc when the --image is obviously not an alias